outline.md

```diff
@@ -304,6 +311,19 @@
   - prove isolation
 - communicating
   - opening issues
+- logging
```

Comment:
another cut paste from the previous 2 days of troubleshooting and can be cut down